### DISCUSSION TITLE
Share via Bluesky GET request URL

### DISCUSSION THREAD

**Discussion by nikiluk** [bluesky-social/atproto#1671] [General Discussion]
Hi! Nikita here, a Founder of Open Ethics Initiative. We'd like to integrate share via Bluesky button on our manifesto page [https://openethics.ai/manifesto](https://openethics.ai/manifesto). 

Could you please specify the URL shape for the GET request to use for the button. Perhaps you already have the integration? Thanks

**Comment by pfrazee** [comment]
Unfortunately we don't support this yet, but I've created an issue to track it https://github.com/bluesky-social/social-app/issues/1540
